be/src/storage/lake/metacache.cpp

```diff
@@ -0,0 +1,238 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
```

Comment:
All variables and methods in this file are moved from TabletManager, no newly added variable or method